pkgs/development/python-modules/crc/default.nix

```
@@ -0,0 +1,22 @@
+{ lib
+, buildPythonPackage
+, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "crc";
+  version = "4.2.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-W1ElEk1127TSXsEHvaU/32zapdY8dH82CWdBNY9r2j4=";
+  };
+
```

```suggestion
  pythonImportsCheck = [
    "crc"
  ];
```

Trying to import the module will also reveal potential issues.